dags/resources/stages/sessions/sqls/root_call_traces.sql

```diff
@@ -0,0 +1,62 @@
+CREATE OR REPLACE TABLE `{destination_project_id}.{temp_dataset_name}.stage_root_call_traces_{ds_no_dashes}`
```

Comment:
temp_dataset_name is in the project where Composer cluster is running. Instead of `{destination_project_id}.{temp_dataset_name}.stage_root_call_traces_{ds_no_dashes}` it should be `{temp_dataset_name}.stage_root_call_traces_{ds_no_dashes}`